.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,166 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "awaiting triage"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Please describe your issue
+    validations:
+      required: true
+  - type: dropdown
+    id: priority
+    attributes:
+      label: Issue Priority
+      description: What priority is this bug?
+      options:
+        - "Priority: 0"
+        - "Priority: 1"
+        - "Priority: 2"
+        - "Priority: 3"
+        - "Priority: 4"
```

Comment:
@kennknowles what do you think about the priorities? 0-4? or some clarification of what each means? (try out here: https://github.com/damccorm/beam-issue-templates/issues/new/choose)

Reply:
Let's rename this to `issue-priorities` https://beam.apache.org/contribute/jira-priorities/

Reply:
I don't think P4 is helpful, but if you have a new way to distinguish P3 and P4 we could add it.

Reply:
> Let's rename this to issue-priorities https://beam.apache.org/contribute/jira-priorities/

I'm a little confused - are you suggesting changing the template, or just renaming the jira priorities page? I have the rename in a larger doc update I'm working on I think - https://github.com/apache/beam/pull/17594 - if not I will add it.

Reply:
I cut p4